### PR TITLE
Add operator architectures explicitly in Makefile

### PR DESCRIPTION
**Pull request by msugakov**
## Description

This follows up on #3163. That PR introduces some code which is for downstream to enable multiple architectures there. The issue is the code is never called upstream and so we're risking it gets rusty or may even get removed as unnecessary.
Having a code that always works is better than having conditional code that works rarely.

Also couple other tiny maintenance changes (see commit descriptions).

## Checklist
- [x] Investigated and inspected CI test results
- ~~[ ] Unit test and regression tests added~~
- ~~[ ] Evaluated and added CHANGELOG entry if required~~
- ~~[ ] Determined and documented upgrade steps~~
- ~~[ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~~

## Testing Performed

- Hoping for CI to cover it.
- Ran `make bundle bundle-build` locally and checked tat `build/bundle/manifests/rhacs-operator.clusterserviceversion.yaml` has `amd64` architecture label.

### CODE REVIEW

**Review by misberner — APPROVED** [review]
👍

File: operator/bundle_helpers/patch-csv.py [hunk 1]
```diff
@@ -70,6 +70,8 @@ def patch_csv(csv_doc, version, operator_image, first_version, no_related_images
     csv_doc["metadata"]["annotations"]["olm.skipRange"] = f'>= {x}.{y-1}.0 < {version}'
 
     # multi-arch
+    if "labels" not in csv_doc["metadata"]:
+        csv_doc["metadata"]["labels"] = {}
```
Comment (yselkowitz) [inline operator/bundle_helpers/patch-csv.py:74]: Why is this necessary?

File: operator/bundle_helpers/patch-csv.py [hunk 2]
```diff
@@ -96,7 +98,7 @@ def parse_args():
                         nargs='+', help='Replacement directives for the RBAC proxy image',
                         default=[])
     parser.add_argument("--add-supported-arch", action='append', required=False,
-                        help='Add label to CSV for an additional operator (may be passed multiple times)',
+                        help='Enable additional operator architecture via CSV labels (may be passed multiple times)',
```
Comment (yselkowitz) [inline operator/bundle_helpers/patch-csv.py:101]: "additional" doesn't seem to make sense if we're starting without any.  (Note that if no such labels are declared, then `amd64` is implied, but once *any* flags are, only those specified are enabled.)

File: operator/bundle_helpers/patch-csv.py (hunk 1 above)
Comment (msugakov) [reply operator/bundle_helpers/patch-csv.py:74]: Because `labels` key does not exist in `metadata` map since I removed default `amd64` there.

**Review by yselkowitz — APPROVED** [review]
